Pipfile

```diff
@@ -59,13 +59,16 @@ zxing-cpp = {version = "*", platform_machine = "== 'x86_64'"}
 # Packages locked due to issues (try to check if these are fixed in a release every so often)
 #
 # Pin this until piwheels is building 1.9 (see https://www.piwheels.org/project/scipy/)
-scipy = "==1.8.1"
+scipy = "~=1.10.0"
 # v4 brings in extra dependencies for features not used here
 reportlab = "==3.6.12"
 # Pin these until piwheels is building a newer version (see https://www.piwheels.org/project/{package}/)
 cryptography = "==40.0.1"
 pikepdf = "==7.2.0"
 pillow = "==9.5.0"
+# Social authentification
```

Comment:
Confusingly, the correct way to add a single package, without updating many others, is `pipenv upgrade social-auth-core`.  This will not remove the libraries Python 3.8 requires.

Reply:
I will try this tomorrow, with a python 3.8

Reply:
After searching for a distribution where I could install 3.8, I discovered that all the major distributions already use 3.10 or 3.11 (alpine, ubuntu, debian, fedora, centos8stream, ...). And for older distribution release, most of them uses python 3.8 versions subject to CVE / already deprecated (If you want to have secure version, you should use 3.8.17, but only fedora, freebsd, gentoo and nix have this version)...
For this kind of software, where sensitive information is stored, I think it's important to keep everything up to date (especially the system itself), so you should really think about dropping armv7 support, or perhaps disabling scipy-related features (I guess "auto matching" feature?).

Reply:
These files are properly locked without breaking changes: https://gist.github.com/stumpylog/e5b7cb51ee9ae83b27e0e70e3b8362ff